src/cljs/rems/navbar.cljs

```diff
@@ -80,13 +84,16 @@
      [language-switcher]]))
 
 (defn navbar-normal [identity]
-  [:nav.navbar-flex
-   [:div.navbar.navbar-expand-sm.flex-fill
-    [:button.navbar-toggler
-     {:type :button :data-toggle "collapse" :data-target "#small-navbar"}
-     "\u2630"]
-    [navbar-items :div#big-navbar.collapse.navbar-collapse.mr-3 identity]]
-   [:div.navbar [user-widget (:user identity)]]])
+  (let [theme @(rf/subscribe [:theme])]
+    [:nav.navbar-flex
+     [:div.navbar.navbar-expand-sm.flex-fill
+      (when-not (str/blank? (:logo-name-navigation theme))
```

Comment:
We can make this simpler with
```clj
(when (:logo-name-navigation theme)
  ...)
```

But also notice that this would be wrong because one can specify `:logo-name-fi-navigation` and not use `:logo-name-navigation`? See what the `styles.clj` does with the `get-theme-attribute`.

Reply:
right, I did not take that one into account. this gets pretty complicated then.

Reply:
We can solve it with a simple utility function that both the `styles.clj` and frontend files use.